src/opentelemetry-core.module.ts

```diff
@@ -102,7 +101,8 @@ export class OpenTelemetryCoreModule implements OnApplicationBootstrap {
     const meterProvider = metrics.getMeterProvider();
 
     if (defaultMetrics) {
-      nodeMetrics(meterProvider);
```

Comment:
Removed the old like out this line out, as it fails with error ,
      // ``` TypeError: nodeMetrics is not a function ```
    
This I faced, while doing local dev environment test by directly importing the `nestjs-otel` using `git+ssh`.

Reply:
Makes sense, it is probably because your project has `esModuleInterop` set to false or not set in your tsconfig.json

Reply:
Actually, `esModuleInterop` is set to `true` in the apps that were pulling in this lib. But still this kept on occurring, just could not figure out why.